### PR TITLE
Refactor Swagger OperationId and Record Name generation

### LINKED ISSUE #289: Refactor OperationId in swagger contract
**Description:**

- [x] Regenerate OperationId for client name by removing special characters and adding Camal letters.
- [ ] Refactor OperationId generated function with checking duplicates 

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by lnash94**
## Purpose
This PR fix special characters in function name and record name 
> Fix #289 
Fix [https://github.com/wso2-enterprise/choreo/issues/5092](https://github.com/wso2-enterprise/choreo/issues/5092)
Fix [https://github.com/wso2-enterprise/choreo/issues/5064](https://github.com/wso2-enterprise/choreo/issues/5064)

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java [hunk 1]
```diff
@@ -308,6 +309,44 @@ public static String escapeIdentifier(String identifier) {
...
+    /**
+     * Generate operationId by removing special characters.
+     *
+     * @param identifier input function name, record name or operation Id
+     * @return string with new generated name
+     */
+    public static String getValidName(String identifier, boolean isSchemaName) {
+        if (!identifier.matches("\\b[a-zA-Z][a-zA-Z0-9]*\\b") && !identifier.matches("\\b[0-9]*\\b")) {
+            String[] split = identifier.split(GeneratorConstants.ESCAPE_PATTERN);
+            StringBuilder operationName = new StringBuilder();
```
Comment (indikasampath2000) [inline openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java:321]: `operatoinName` must be `validName` right? We are not getting a valid name only for operation id, also for schema name as well.

File: openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java [hunk 2]
```diff
@@ -308,6 +309,44 @@ public static String escapeIdentifier(String identifier) {
         return identifier;
     }
 
+    /**
+     * Generate operationId by removing special characters.
+     *
+     * @param identifier input function name, record name or operation Id
+     * @return string with new generated name
+     */
+    public static String getValidName(String identifier, boolean isSchemaName) {
```
Comment (indikasampath2000) [inline openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java:318]: `isSchemaName` should be `isSchema`

File: openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java [hunk 3]
```diff
@@ -308,6 +309,44 @@ public static String escapeIdentifier(String identifier) {
...
+        }
+    }
+
+    /**
+     * Check the given recordName is valid name.
+     *
+     * @param recordName - String record name
+     * @return           - boolean value
+     */
+    public static boolean isValidRecordName(String recordName) {
```
Comment (indikasampath2000) [inline openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java:346]: `isValidRecordName` should be `isValidSchemaName`

File: openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java (hunk 1 above)
Comment (lnash94) [reply openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java:321]: yeah, will change with `validName`

Comment (lnash94) [reply openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java]: resovled with [https://github.com/ballerina-platform/ballerina-openapi/pull/291/commits/0479016e743216e05b5282cfa2d4890bf19d33cb](https://github.com/ballerina-platform/ballerina-openapi/pull/291/commits/0479016e743216e05b5282cfa2d4890bf19d33cb)

**Pull request merged**